pkg/diagnose/cni.go

```diff
@@ -228,7 +228,8 @@ func getSpecBool(pool unstructured.Unstructured, key string) (bool, error) {
 	}
 
 	if !found {
-		return false, fmt.Errorf("%s status not found for IPPool %q", key, pool.GetName())
+		// for bool type, not found is considered isDisabled
+		return true, nil
```

Comment:
This should be false.

```suggestion
		// for bool type, not found is considered false
		return false, nil
```

Reply:
Thanks @tpantelis for the review. Updated.